Lection 3/task 3.1/ReportApp/Interfaces/IShop/IShopConfigurationLoader.cs

```diff
@@ -0,0 +1,9 @@
+﻿using ReportApp.Models.Shop;
+
+namespace ReportApp.Interfaces.IShop;
+
+public interface IShopConfigurationLoader
+{
+    public ShopReportConfiguration LoadFromFile(string path);
```

Comment:
As loadFromFile is very specific for each type of report config and won't be used outside of the model - keep it in model